README.md

```diff
@@ -43,10 +43,15 @@ Below is the list of variables you can redefine in your playbook to customize st
 | `st2_auth_password`      | `testp`       | Password used by StackStorm standalone authentication.
 | `st2_save_credentials`   | `yes`         | Save credentials for local CLI in `/root/.st2/config` file.
 | **st2mistral**
-| `st2mistral_version`     | `latest`      | st2mistral version to install. Use latest `latest` to get automatic updates or pin it to numeric version like `2.1.1`.
+| `st2mistral_version`     | `latest`      | st2mistral version to install. Use latest `latest` to get automatic updates or pin it to numeric version like `2.2.0`.
 | `st2mistral_db`          | `mistral`     | PostgreSQL DB name for Mistral.
 | `st2mistral_db_username` | `mistral`     | PostgreSQL DB user for Mistral.
 | `st2mistral_db_password` | `StackStorm`  | PostgreSQL DB password for Mistral.
+| **st2chatops**               | Default hubot adapter: 'slack'
+| `st2chatops_version`     | `latest`      | st2mistral version to install. Use `latest` to get automatic updates or pin it to numeric version like `2.2.0`.
+| `st2chatops_ST2_API_KEY` | `CHANGE-ME-PLEASE`      | st2 API key, that can be generated using `st2 apikey create -k` in a task.
```

Comment:
The problem with this approach is that we need a new var for every setting in `st2chatops.env`. And if we introduce new setting (like recent Mattermost support https://github.com/StackStorm/st2chatops/pull/65/files) it will force us to edit `ansible-st2` as well.

The better approach is to have `st2chatops_config` hash/dict which will allow us to configure *ANY* env variable in `st2chatops.env`.
Take a look at my comment, we need the following syntax to work: https://github.com/StackStorm/ansible-st2/pull/125#issuecomment-284414479

```sh
# hash to configure any value in st2chatops.env
# https://github.com/StackStorm/st2chatops/blob/master/st2chatops.env
st2chatops_config:
  ST2_API_KEY: 123asd456
  HUBOT_ADAPTER: slack
  HUBOT_SLACK_TOKEN: xoxb-CHANGE-ME-PLEASE
  # add here any other var
```

At a high level we just have `{{ st2chatops_config }}` var to rule all config settings.
Similar approach is implemented in https://github.com/StackStorm/ansible-st2/pull/121/files